Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -214,7 +214,7 @@ namespace O3DE::ProjectManager
 
         m_gemDisplayName = new FormLineEditWidget(
             tr("Gem Display name*"), "", tr("The name displayed in the Gem Catalog"), tr("A gem display name is required."));
-        m_gemDisplayName->lineEdit()->setValidator(new QRegularExpressionValidator(QRegularExpression("\\w+"), this));
+        m_gemDisplayName->lineEdit()->setValidator(new QRegularExpressionValidator(QRegularExpression("( |\\w)+"), this));
```

Comment:
Why not use `\s`?

Reply:
Main concern is that for this field we did not want to unnecessarily allow other whitespace characters like newlines. As such I chose to only include spaces specifically. 

It's meant to be a single line, so allowing for the possibility of newlines may cause text rendering issues later on in some unintended way.